flexget/plugins/notifiers/email.py

```diff
@@ -104,56 +101,47 @@ class EmailNotifier(object):
         'additionalProperties': False,
     }
 
-    def notify(self, to, message, title, smtp_host, smtp_port, smtp_username=None, smtp_password=None, smtp_tls=None,
-               smtp_ssl=None, html=None, **kwargs):
+    def notify(self, title, message, config):
```

Comment:
But... it wasn't passing in literal params, it was passing in a config dict, with other stuff added to it, which was being keyword argument expanded. This way we are passing in our actual literal params, which are the ones generated by the notification system, and the config, as specified and validated by the schema of the plugin, like we do for our other plugin methods.